azure-templates/pre-job-steps.yml

```diff
@@ -0,0 +1,73 @@
+parameters:
+  - name: customDeviceRepoName
+    type: string
+  - name: lvVersion
+    type: string
+  - name: bitness
+    type: string
+  - name: buildTools
+    type: string
+    default: niveristand-custom-device-build-tools
+
+steps:
+###########  Configure Job-wide Variables  ###########
+  # --- lvPath, architecture ---
+  - ${{ if eq( parameters.bitness, '32-bit' )}}:
+    - powershell: |
+        Write-Host "##vso[task.setvariable variable=lvPath]C:\Program Files (x86)\National Instruments\LabVIEW ${{ parameters.lvVersion }}"
+        Write-Host "##vso[task.setvariable variable=architecture]x86"
+        Write-Host "##vso[task.setvariable variable=nipkgx86suffix]-x86"
+        Write-Host "##vso[task.setvariable variable=nipkgx64suffix]"
+      displayName: Use 32-bit LabVIEW
+
+  - ${{ if eq( parameters.bitness, '64-bit' )}}:
+    - powershell: |
+        Write-Host "##vso[task.setvariable variable=lvPath]C:\Program Files\National Instruments\LabVIEW ${{ parameters.lvVersion }}"
+        Write-Host "##vso[task.setvariable variable=architecture]x64"
+        Write-Host "##vso[task.setvariable variable=nipkgx86suffix]"
+        Write-Host "##vso[task.setvariable variable=nipkgx64suffix]64"
+      displayName: Use 64-bit LabVIEW
+
+  # --- lvConfigVersion (.NET Assembly Version) ---
+  - ${{ if eq( parameters.lvVersion, '2020' )}}:
+    - powershell: |
+        Write-Host "##vso[task.setvariable variable=lvConfigVersion]8.0.0.0"
+        Write-Host '##vso[task.setvariable variable=shortLvVersion]20'
+        Write-Host '##vso[task.setvariable variable=releaseVersion]20.0.0'
+        Write-Host '##vso[task.setvariable variable=quarterlyReleaseVersion]$(releaseVersion)'
+      displayName: Use .NET Assembly 8.0.0.0
+
+  - ${{ if eq( parameters.lvVersion, '2021' )}}:
+    - powershell: |
+        Write-Host "##vso[task.setvariable variable=lvConfigVersion]9.0.0.0"
+        Write-Host '##vso[task.setvariable variable=shortLvVersion]21'
+        Write-Host '##vso[task.setvariable variable=releaseVersion]21.0.0'
+        Write-Host '##vso[task.setvariable variable=quarterlyReleaseVersion]$(releaseVersion)'
+      displayName: Use .NET Assembly 9.0.0.0
+
+  - ${{ if eq( parameters.lvVersion, '2023' )}}:
+    - powershell: |
+        Write-Host "##vso[task.setvariable variable=lvConfigVersion]10.0.0.0"
+        Write-Host '##vso[task.setvariable variable=shortLvVersion]23'
+        Write-Host '##vso[task.setvariable variable=releaseVersion]23.0.0'
+        Write-Host '##vso[task.setvariable variable=quarterlyReleaseVersion]2023 Q1'
```

Comment:
The release version is tied to the version of the releasing package, not the LabVIEW version.

For example, we are working on a 23.0 release (quarterly version 2023 Q1), but it will have LabVIEW builds in 2020, 2021, and 2023.

Reply:
moved to be its own parameters in commit 5